### ISSUE TITLE
While on Live Share, host computer's cursor is moved to remote's cursor when docstring is auto-inserted

### ISSUE THREAD

**Issue by aely-aronoff** [microsoft/pylance-release#1592]
TL;DR On Live Share, if a remote machine types three quotes, a matching set of three quotes are inserted, but the host machine's cursor gets moved to the place where the quotes are inserted. Additionally, when the host and remote machines have different files open, the remote developer's closing triple quotes don't get inserted. 

This issue appeared in the most recent update, after #1583 was introduced in 2021.7.5.

## Environment data

Host machine:

* VS Code version: Code 1.57.1 (507ce72a4466fbb27b715c3722558bb15afa9f48, 2021-06-17T13:28:07.755Z)
* Extension version (available under the Extensions sidebar): 2021.7.5
* OS and version: Windows_NT x64 10.0.19042
* Python version (& distribution if applicable, e.g. Anaconda): Python 3.9.1 64-bit
* Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A
* Relevant/affected Python-related VS Code extensions and their versions: vsliveshare v1.0.4583
* Value of the python.languageServer setting: default, which is I think pylance

Remote machine:

* VS Code version: Code 1.58.2
* Extension version (available under the Extensions sidebar): 2021.7.5
* OS and version: 11.1 macOS Big Sur
* Relevant/affected Python-related VS Code extensions and their versions: vsliveshare v1.0.4583

## Expected behaviour

When the remote developer types three quotes for a docstring, three matching quotes should appear at the remote developer's cursor, without anything happening to the position of the host machine's cursor and irrespective of what file the host machine has open.

## Actual behaviour

If the host and remote machines both have the same file open, the host's cursor is moved to the remote machine's cursor when the remote developer types three quotes.

If they have different files open, quotes aren't auto-inserted at all at the remote's cursor.

**Comment by jakebailey** [comment]
I haven't had a chance to set up LiveShare to debug this one, but I've fixed #1716, which should disable triple quote auto-closing when there are multiple selections, which may apply to LiveShare. I'm thinking that each cursor is its own selection, where selection zero (whose use I removed when fixing #1583) is the host machine's. Disabling this when we have more than one cursor may also apply to LiveShare for that reason.

**Comment by heejaechang** [comment]
we might just disable auto triple quote insertion on liveshare.

**Comment by heejaechang** [comment]
old issue. please reopen if it is still an issue.
